lms/templates/staff_problem_info.html

```diff
@@ -6,7 +6,7 @@
 
 ## The JS for this is defined in xqa_interface.html
 ${block_content}
-%if location.category in ['problem','video','html','combinedopenended','graphical_slider_tool', 'library_content']:
+%if location.category in ['problem','video','html','combinedopenended','library_content']:
```

Comment:
Not really related to your PR, just wanted to let you (and others) know that this is related to the work in a different PR: https://github.com/edx/edx-platform/pull/10970